tmc-langs-r/src/test/java/fi/helsinki/cs/tmc/langs/r/RPluginTest.java

```diff
@@ -0,0 +1,203 @@
+package fi.helsinki.cs.tmc.langs.r;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import fi.helsinki.cs.tmc.langs.domain.RunResult;
+import fi.helsinki.cs.tmc.langs.domain.TestDesc;
+import fi.helsinki.cs.tmc.langs.domain.TestResult;
+import fi.helsinki.cs.tmc.langs.io.StudentFilePolicy;
+import fi.helsinki.cs.tmc.langs.utils.TestUtils;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class RPluginTest {
+
+    private RPlugin plugin;
+
+    private Path simpleAllTestsPassProject;
+    private Path simpleSomeTestsFailProject;
+    private Path simpleSourceCodeErrorProject;
+
+    @Before
+    public void setUp() {
+        plugin = new RPlugin();
+
+        simpleAllTestsPassProject = TestUtils.getPath(getClass(),
+                "simple_all_tests_pass");
+        simpleSomeTestsFailProject = TestUtils.getPath(getClass(),
+                "simple_some_tests_fail");
+        simpleSourceCodeErrorProject = TestUtils.getPath(getClass(),
+                "simple_source_code_error");
+    }
+
+    @After
+    public void tearDown() {
+        removeAvailablePointsJson(simpleAllTestsPassProject);
+        removeResultsJson(simpleAllTestsPassProject);
+        removeResultsJson(simpleSomeTestsFailProject);
+        removeResultsJson(simpleSourceCodeErrorProject);
+    }
+
+    private void removeResultsJson(Path projectPath) {
+        File resultsJson = new File(projectPath.toAbsolutePath().toString()
+                + "/.results.json");
+        resultsJson.delete();
+    }
+
+    private void removeAvailablePointsJson(Path projectPath) {
+        File availablePointsJson = new File(projectPath.toAbsolutePath().toString()
+                + "/.available_points.json");
+        availablePointsJson.delete();
+    }
+
+    private void testResultAsExpected(TestResult result, boolean successful, String name,
+                                      String[] points) {
+        assertEquals(successful, result.isSuccessful());
+        assertEquals(name, result.getName());
+        assertArrayEquals(points, result.points.toArray());
+    }
+
+    @Test
+    public void testGetTestCommand() {
```

Comment:
What is the value of this test? This looks like you're testing whether the getter you're calling is written exactly the same way as its parallel implementation in this test method.